services/collaboration/README.md

```diff
@@ -114,68 +115,82 @@ curl -X GET http://localhost:8080/api/collaboration/room/user/rooms \
 {
   "status": "Success",
   "data": [
-    "6721a64b0c4d990bc0feee4c"
+    "6724e9d892fb3e9f04c2e280"
   ]
 }
 ```
 
 ---
 
-## Get Room by Room ID
+## Get Rooms by User ID, Room Status, and User's isForfeit status
 
-This endpoint retrieves the details of a room by its room ID.
+This endpoint retrieves the details of rooms associated with the authenticated user, filtered by the specified room
+status.
 
 - **HTTP Method**: `GET`
-- **Endpoint**: `/api/collaboration/room/{roomId}`
+- **Endpoint**: `/user/rooms/roomStatus/{room_status}/isForfeit/{isForfeit}`
 
 ### Authorization
 
 This endpoint requires a valid JWT token in the Authorization header.
 
 ### Parameters:
 
-- `roomId` (Required) - The ID of the room to retrieve.
+- `room_status` (Required) - The status of the room to filter by (`true` for open rooms, `false` for closed rooms).
```

Comment:
Just a nit, `room_status` should follow the camelCase that seems to be used in the rest of the README instead of snake_case.